java/src/jmri/NamedBeanBundle.properties

```diff
@@ -36,6 +36,9 @@ BeanNamePortal       = Portal
 EntryExit            = Entry Exit
 FastClock            = Fast Clock
 BeanName             = Named Bean
+# use the MakeLabel key to add a localized interpunction after the above (or any other) item
```

Comment:
I totally agree there is a balance to be struck. But always having two of everything just to prevent having ... and : in the wrong places (i.e. in a window title because the menu has the same name) is what this implies. Also, the downside of having context when writing is missing a view of the whole program and introducing all sorts of descrepencies between terms and interface elements, calling them Remove in one place and Delete in another, or OK vs. Okay. Regarding numbers, in JMRI things are a bit simpler than these rules imply, and just like a variable index that counts from 1 to n, using A and B in all languages helps us talk about the program across Locales. But sure, the MakeLabel key does seem a bit too abstract to fix this.